rust/connlib/tunnel/src/tests.rs

```diff
@@ -405,7 +410,9 @@ impl TunnelTest {
                             continue;
                         }
 
-                        if let ControlFlow::Break(_) = self.try_handle_gateway(dst, src, &payload) {
+                        if let ControlFlow::Break(_) =
```

Comment:
Can you do `is_break()` here like in that other PR?

Reply:
I will as part of resolving the conflicts :)